app/views/users/edit.html.erb

```diff
@@ -59,7 +41,31 @@
       </div>
     </div>
 
-    <div class="crayons-layout__content">
+    <div class="crayons-layout__content mt-3">
+      <div class="crayons-page-header">
+        <% if @tab == "organization" && @organizations.present? %>
+          <% if !@organization&.new_record? %>
+            <section class="flex m:flex-row justify-between w-100 mb-4 flex-col">
+              <% options = @organizations.pluck(:slug, :id).each do |arr| %>
+                <% arr[0] = "@#{arr[0]}" %>
+                <% arr[1] = "/settings/organization/#{arr[1]}" %>
+              <% end %>
+              <%= tag.select(options_for_select(options, "/settings/organization/#{params[:org_id]}"), class: "crayons-select lh-tight m:w-25 align-text-top m:mb-0 m:w-max w-100 mb-2", onchange: "window.location.href=this.value") %>
+              <%# For example: %>
+              <%# <option value="/settings/organization/1>@devteam</option> %>
+              <%# <option value="/settings/organization/new" selected>New Organization</option> %>
+              <%= button_to url_for("/settings/organization/new"), class: "crayons-btn m:w-max w-100 follow-action-button showing", method: :get do %>
```

Comment:
`follow-action-buttons` are usually used for "follow" buttons and so i dont think we need this class here. 

```suggestion
              <%= button_to url_for("/settings/organization/new"), class: "crayons-btn m:w-max w-100", method: :get do %>
```